modules/data-streams/src/test/java/org/elasticsearch/datastreams/action/TransportGetDataStreamsActionTests.java

```diff
@@ -49,82 +45,6 @@ public class TransportGetDataStreamsActionTests extends ESTestCase {
         ClusterSettings.createBuiltInClusterSettings()
     );
 
-    public void testGetDataStream() {
```

Comment:
Moved these tests to `DataStreamsActionUtilTests` since we are using that method now to get the names.